Assets/Mirage/Runtime/InterestManagement/InterestManager.cs

```diff
@@ -265,8 +264,11 @@ private void Observers(NetworkIdentity identity)
                 }
             }
 
+            if (!foundList)
```

Comment:
```suggestion
            if (inSystemsCount > 0)
```

Reply:
Cant use that count because global system does not register to the list. I am checking for global objects incase someone adds a aoi sytem during runtime it will auto remove from global. Only issue here is you cant go in reverse from aoi back to global but I dont think that's a huge issue. We can add that later if we want

Reply:
Makes sense!